open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventConsumer.java

```diff
@@ -3,14 +3,14 @@
 package org.odpi.openmetadata.adapters.eventbus.topic.kafka;
 
 import java.time.Duration;
-import java.util.ArrayDeque;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Properties;
 import java.util.Queue;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;

```

Comment:
why change a specific import to a wildcard ?

Reply:
This idea was made automatically I just did not pay attention. But resole it is easily.

Reply:
Good spot @wbittles - certainly we should not use wildcards.

Reply:
Yes, I removed it just I am vocation now:)